multiqc/modules/whatshap/whatshap.py

```diff
@@ -0,0 +1,319 @@
+        if len(sample_stats) == 1:
+            summary_field = list(sample_stats)[0]
+        else:
+            summary_field = "ALL"
+        return summary_field
+
+    def whatshap_add_general_stats(self):
+        """Add WhatsHap stats to the general statistics table"""
+
+        # Store the configuration for each column, this is also used to
+        # determine which columns to add to the general statistics table
+        # https://whatshap.readthedocs.io/en/latest/guide.html#the-tsv-statistics-format
+        general_stats_headers = OrderedDict(
+            [
+                (
+                    "variants",
+                    {
+                        "id": "variants",
+                        "title": "Input Variants",
+                        "description": "Number of biallelic variants in the input VCF, but "
+                        "excluding any non-SNV variants if --only-snvs was "
+                        "used",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "heterozygous_variants",
+                    {
+                        "id": "heterozygous_variants",
+                        "title": "Heterozygous Variants",
+                        "description": "The number of biallelic, heterozygous variants in "
+                        "the input VCF. This is a subset of Input Variants.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "heterozygous_snvs",
+                    {
+                        "id": "heterozygous_snvs",
+                        "title": "Heterozygous SNVs",
+                        "description": "The number of biallelic, heterozygous SNVs in the "
+                        "input VCF. This is a subset of Heterozygous "
+                        "Variants.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "unphased",
+                    {
+                        "id": "unphased",
+                        "title": "Unphased Variants",
+                        "description": "The number of biallelic, heterozygous variants that "
+                        "are not marked as phased in the input VCF. This "
+                        "is a subset of heterozygous_variants.",
+                        "hidden": False,
+                    },
+                ),
+                (
+                    "phased",
+                    {
+                        "id": "phased",
+                        "title": "Phased Variants",
+                        "description": "The number of biallelic, heterozygous variants that "
+                        "are marked as phased in the input VCF. This is "
+                        "a subset of heterozygous_variants. Also, phased + "
+                        "unphased = heterozygous_variants.",
```

Comment:
I’ve just fixed this in the documentation:
```suggestion
                        "unphased + singletons = heterozygous_variants.",
```

Reply:
According to the whatshap documentation, "phased + unphased = heterozygous_variants". Where did you find that singletons should be included as well?

Reply:
The commit where I changed it is at https://github.com/whatshap/whatshap/commit/60de9f1f0a8cb252e9077d0dbf03b2457ae808e9. The documentation did not reflect this change until a moment ago due to a build failure on Read the Docs, sorry about that.

I made that change because I was working on the `stats` subcommand for a different reason and realized that the description in the documentation was incorrect.